src/app/content/practiceQuestions/hooks/locationChange.spec.ts

```diff
@@ -85,8 +99,46 @@ describe('locationChange', () => {
 
     await hook();
 
+    const section = archiveTreeUtils.findArchiveTreeNodeById(book.tree, shortPage.id) as LinkedArchiveTreeSection;
+
     expect(getSummary).not.toHaveBeenCalled();
     expect(dispatch).not.toHaveBeenCalledWith(receivePracticeQuestionsSummary(mockSummaryResponse));
+    expect(dispatch).toHaveBeenCalledWith(setSelectedSection(section));
+  });
+
+  it('only fetch practice questions if section is undefined', async() => {
```

Comment:
I'm not sure what this test is about? When we are fetching PQ only when `section` is undefined in `locationChange`?

Reply:
i still don't get it, the only `section` i can see in `locationChange` is this line:
```
const section = findArchiveTreeNodeById(book.tree, page.id);
```
but we are not fetching anything related to this section

maybe the test name should be `do not set selected section if it is undefined`?